jac-uk/admin
# #2121 Ensure the Candidate Page Accounts For Post 01/04/2023 Social Mobility Fields

## What's included?
Using latest jac-kit which has added checks so that post 0104 answers are reflected in the candidate details page. Prioritising post 0104 answers over pre-0104 ones.

Closes #2121 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Ensure the candidate page shows post 01-04-2023 candidate questions.

**Note the following:**

**Post 01-04-2023** questions are:

- Attended state or fee-paying school between 11-16
- Occupation of childhood earner
- Parents attended university

**Pre 01-04-2023** questions are:

- Attended state or fee-paying school
- Attended Oxbridge universities
- First generation to go to university

**Test 1 - Post 01-04-2023 questions apply**
- Login
- Go to this page: https://jac-admin-develop--pr2157-featire-2121-candida-jc7dqfev.web.app/candidates/KnTFmjXqtDchIKLGuwkTC5Om4982
- Ensure the post 01 04 2023 questions are displayed

**Test 2 - Pre 01-04-2023 questions apply**
- Go to this page: https://jac-admin-develop--pr2157-featire-2121-candida-jc7dqfev.web.app/candidates/DdvJcPCIYhgpZDFNhahO5nNvMMA2
- Ensure the pre 01 04 2023 questions are displayed

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
